app/src/main/java/nu/yona/app/YonaApplication.java

```diff
@@ -36,6 +38,12 @@
 	private static EventChangeManager eventChangeManager;
 	private Tracker tracker;
 
+
+	public static SharedPreferences sharedAppPreferences;
```

Comment:
Please wrap these three in accessor methods (e.g. ``getSharedAppPreferences()``) and make the variables private.  Modifiable public variables is considered a bad practice.

Reply:
Done. Update pushed.